README.md

```diff
@@ -119,25 +119,25 @@ Ao iniciar um projeto, deve-se mostrar para o Git que esta pasta é um repositó
 (7) λ ls
 ```
 
-Ao inicializar o Git (5), é criada uma pasta oculta com o nome .git, sendo possível enxergá-la com uma opção -a para o código dir (6) ou utilizar o (7). Também é criada uma branch (que vem com o nome *default* `master`). Se no caso a branch padrão do GitHub estiver com outro nome, basta altrar no próprio GitHub ou alterar a do Git local com os seguintes comandos:
+Ao inicializar o Git (5), é criada uma pasta oculta com o nome .git, sendo possível enxergá-la com uma opção -a para o código dir (6) ou utilizar o (7). Também é criada uma *branch* (que vem com o nome *default* `master`). Se no caso a *branch* padrão do GitHub estiver com outro nome, basta altrar no próprio GitHub ou alterar a do Git local com os seguintes comandos:
 
 ```
-(8.1) λ git branch
-(8.2) λ git branch --list
-(8.3) λ git branch -m [ANTIGO] [NOVO]
-(8.4) λ git branch -a
+(8.1) λ git *branch*
```

Comment:
Exemplo de uma formatação errada do itálico, pois o código está dentro de um **code block** (```)